libs/MeshKernelApi/tests/src/CurvilinearGridTests.cpp

```diff
@@ -995,6 +995,9 @@ class CurvilinearLocationIndexTests : public ::testing::TestWithParam<std::tuple
         return {
```

Comment:
You can add `using namespace meshkernel;` at the beginning of this source and remove all the `meshkernel::` namespaces. Easier on the eye.